annarosepaloma/sass-annarosepaloma
# Add npm script for JS production build and JS watch script

# Related Tickets
https://codeandtheory.atlassian.net/browse/FE-15

# Description
This PR adds the npm scripts for JS production build and JS watch script.

# Validation

- Run `npm start` to compile the development build and watch for changes in real-time.
- Run `npm run prod` for production build.
- After running `npm run prod`, the CSS and JS files on `dist` folder should be minified.
